src/main/java/org/jenkinsci/plugins/pubsub/EventProps.java

```diff
@@ -116,6 +135,10 @@
          * Job run status/result.
          */
         job_run_status,
+        /**
+         * Job run SCM commit Id, if relevant.
+         */
+        job_run_commitId,
```

Comment:
@tfennelly watch out - we just ran into a stack of problems calculating this one - if you are feeding through the commit Id make sure its the right one. See https://github.com/jenkinsci/blueocean-plugin/pull/1010 ping @vivek